test/spec/provider/camunda/element-templates/ReplaceBehaviorSpec.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+var bootstrapModeler = require('../../../../helper').bootstrapModeler,
+    camundaModdlePackage = require('camunda-bpmn-moddle/resources/camunda'),
+    coreModule = require('bpmn-js/lib/core').default,
+    inject = require('../../../../helper').inject,
+    modelingModule = require('bpmn-js/lib/features/modeling').default,
+    propertiesPanelModule = require('lib'),
+    propertiesProviderModule = require('lib/provider/camunda'),
+    replaceModule = require('bpmn-js/lib/features/replace').default;
+
+describe('element-templates - replace behavior', function() {
+  var testModules = [
+    coreModule,
+    modelingModule,
+    replaceModule,
+    propertiesPanelModule,
+    propertiesProviderModule
+  ];
+
+  var diagramXML = require('./fixtures/replace-behavior.bpmn');
+
+  beforeEach(bootstrapModeler(diagramXML, {
+    elementTemplates: require('./fixtures/replace-behavior'),
+    modules : testModules,
+    moddleExtensions: { camunda: camundaModdlePackage }
+  }));
+
+  it('should link to an element of a type the template applies to', inject(function(elementRegistry, bpmnReplace) {
+
+    // given
+    var task = elementRegistry.get('task_1');
+    var newElementData = {
+      type: 'bpmn:CallActivity'
+    };
+
+    // when
+    var newElement = bpmnReplace.replaceElement(task, newElementData);
+
+    // then
+    var businessObject = newElement.businessObject;
+    expect(businessObject.modelerTemplate).to.exist;
+  }));
+
+
+  it('should link to a sub-element of a type the template applies to', inject(function(elementRegistry, bpmnReplace) {
+
+    // given
+    var task = elementRegistry.get('task_1');
+    var newElementData = {
+      type: 'bpmn:ServiceTask'
+    };
+
+    // when
+    var newElement = bpmnReplace.replaceElement(task, newElementData);
+
+    // then
+    var businessObject = newElement.businessObject;
+    expect(businessObject.modelerTemplate).to.exist;
+  }));
+
+
+  it('should not link to an element of a type the template doesn\'t apply to', inject(function(elementRegistry, bpmnReplace) {
+
+    // given
+    var task = elementRegistry.get('task_1');
+    var newElementData = {
+      type: 'bpmn:SubProcess'
+    };
+
+    // when
+    var newElement = bpmnReplace.replaceElement(task, newElementData);
+
+    // then
+    var businessObject = newElement.businessObject;
+    expect(businessObject.modelerTemplate).not.to.exist;
+  }));
```

Comment:
Add new line here. As a general rule, we add two new lines between `it` blocks and between setup blocks and `it` blocks.

```javascript
setup();

moreSetup();


describe('...', function() {

  setup();

  moreSetup();


  it('should ...');


  it('should ...');


  describe('...', function() {

    it('should ...');

  });

});
```